### PR TITLE
Update wordlereminder.py

### PR DESCRIPTION
**Pull request by brods**
I thought it would be funny if the wordle reminder gently abused people but I don't know if this is the right way to get it to have a set of responses or what the f is doing in the first message :(

### CODE REVIEW

**Comment by ESloman** [comment]
Haha this is a hilarious idea!

You've got the gist; but take a look at the `dailyvallytask.py` to see how that handles selecting a random message.

We have a defined list of possibles messages (as you've created):

https://github.com/ESloman/bsebot/blob/main/discordbot/tasks/dailyvallytask.py#L26-L39

Note, how each one has a `{role}`. This bit of text gets replaced by the actual role mention later; we'll need to do something similar. Currently, we use f-strings and have `{y_message.author.mention}` in there, this is inserting the username mention inline there and then. We could have a `{mention}` tag and then replace that later on like we do with the valorant tag.

This is how we then select one of those possible answers:

https://github.com/ESloman/bsebot/blob/main/discordbot/tasks/dailyvallytask.py#L120-L121

Note, we'll need to add `import random` at the top of the file somewhere like we do in `dailyvallytask.py`.

Hopefully that helps a bit! Give me a shout if you need more help / have questions.

**Comment by brods** [comment]
OH LAWD WE'RE FAILING LINTING

**Comment by brods** [comment]
FUCK LINTING

File: discordbot/tasks/wordlereminder.py [hunk 1]
```diff
@@ -101,17 +108,16 @@ async def wordle_reminder(self):
...
-            self.logger.info(msg)
-            await y_message.reply(content=msg)
+            self.logger.info(message)
+            await y_message.reply(content=message)
 
     @wordle_reminder.before_loop
     async def before_wordle_reminder(self):
         """
-        Make sure that websocket is open before we starting querying via it.
+        Make sure that websocket is open before we start querying via it.
```
Comment (ESloman) [inline discordbot/tasks/wordlereminder.py:120]: I copy-paste the same boilerplate code I have for a task into _every_ task so I imagine this typo exists in the other task files as well 😂 easy contributions for you eventually!

**Review by ESloman — APPROVED** [review]
lgtm

**Comment by ESloman** [comment]
Okay, I was trying to fix the merge conflict so it could be merged (felt like that shouldn't be left to you 😂) so hopefully that's still fine.

**Review by ESloman — APPROVED** [review]
still lgtm

File: discordbot/tasks/wordlereminder.py (hunk 1 above)
Comment (brods) [reply discordbot/tasks/wordlereminder.py:129]: hahahaha

**Pull request merged**